README.md

```diff
@@ -1,29 +1,29 @@
 # Composite Container-based Application Specification
 
-`/NOO-le-kyul/` (n.) a made-up word meaning ["the mother of all atomic particles"](http://simpsons.wikia.com/wiki/Made-up_words). Sounds like "molecule". But different.
+`/NOO-le-kyul/` (n.) a made-up word meaning ["the mother of all atomic particles"](http://simpsons.wikia.com/wiki/Made-up_words). It is spelled like "molecule", but pronounced like "noodle".
 
```

Comment:
Fair point :)  Perhaps we delete the second sentence entirely and put in an proper IPA pronunciation guide:

\ˈnü-li-ˌkyül\